test/e2e/framework/util.go

```diff
@@ -3835,7 +3844,7 @@ func IssueSSHCommandWithResult(cmd, provider string, node *v1.Node) (*SSHResult,
 	host := ""
 	for _, a := range node.Status.Addresses {
 		if a.Type == v1.NodeExternalIP {
-			host = a.Address + ":22"
+			host = fmt.Sprintf("%v:%v", a.Address, sshPort)
```

Comment:
It is. But using Sprintf vs. string addition is a more common pattern from what I've seen, and I was on that line of code. I can change it back if it matters.